packages/core/src/packager/packager.ts

```diff
@@ -58,8 +58,12 @@ const Yarn = Implementation({
 type Manager = typeof NPM | typeof Yarn;
 
 export function getManager(dir: string): Manager {
+  const topPath = /^[a-z]+:\\$/gi;
```

Comment:
Can you hoist this outside this function and put it in a `const WINDOWS_TOP_PATH`